INSTALL

```diff
@@ -109,7 +109,7 @@ Prepare your system
  2. Install dtrace (to generate provider.h)
     sudo apt-get install systemtap-sdt-dev
  3. Install libdispatch pre-reqs
-    sudo apt-get install libblocksruntime-dev libkqueue-dev libbsd-dev
+    sudo apt-get install libblocksruntime-dev libbsd-dev
```

Comment:
I wouldn't *quite* do that, as I think that it won't fly long term with distributions to not use the system's libkqueue. so I wouldn't touch this, but use pkg-config or some method in configure to check that the system libkqueue is recent enough to have the right fixes instead.